yb-voyager/cmd/import.go

```diff
@@ -71,6 +71,9 @@ func validateImportFlags(cmd *cobra.Command) {
 }
 
 func registerCommonImportFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&tconf.TargetDBType, "target-db-type", "",
+		"type of the target database (oracle, mysql, postgresql)")
```

Comment:
yugabytedb?
after adding yugabytedb to the list, use it in automation tests instead of `postgres`